src/schemas/tokens-info.ts

```diff
@@ -57,6 +57,11 @@ const BaseTokenInfoSchema = Joi.object({
   minimumAppVersionToSwap: Joi.string()
     .pattern(/^\d+\.\d+\.\d+$/)
     .custom(checkMinVersion, 'has a valid version'),
+  maxSupportedVersion: Joi.object({
+    valora: Joi.string()
+      .pattern(/^\d+\.\d+\.\d+$/)
+      .custom(checkMinVersion, 'has a valid version'),
```

Comment:
nit: rename `checkMinVersion` to checkSemverVersion or something and update the functions error message as well.

Reply:
ahh I didn't realize this check was happening, what would the check be for this case? I guess it just needs to be a valid version string but the Joi parsing should handle that, there is no min it can be otherwise (and I guess max is current version but that keeps changing, and not necessarily if we plan on deprecating a token at a certain future version)

Reply:
Ah, I see what that check is doing now. i thought it was doing the regex check.

Reply:
Yea, go ahead and remove the checkMinVersion check from this one.